client/state/form/actions.js

```diff
@@ -10,15 +16,67 @@ export const setFormProperty = ( field, value ) => {
 	};
 };
 
-export const nextStep = ( ) => {
-	return {
-		type: NEXT_STEP,
-	};
-};
-
 export const goToStep = ( stepIndex ) => {
 	return {
 		type: GO_TO_STEP,
 		step: stepIndex,
 	};
 };
+
+export const nextStep = () => ( dispatch, getState, { callbackURL, nonce, formSchema, formLayout } ) => {
```

Comment:
Since this action submits the form, maybe a better name would be `submitStep` since `nextStep` could be casually misinterpreted to mean simply mutating the step in state